docker/Dockerfile-build

```diff
@@ -57,4 +57,14 @@ RUN mkdir -p /tmp/setup-kotlin \
     && sdk install kotlin \
     && echo ". /home/circleci/.sdkman/bin/sdkman-init.sh" >> /home/circleci/.profile \
     && cd ../ \
-    && rm -rf ./setup-kotlin
\ No newline at end of file
+    && rm -rf ./setup-kotlin
+
+RUN mkdir -p /tmp/setup-jna \
+    && cd /tmp/setup-jna \
+    && curl -o jna.jar https://repo1.maven.org/maven2/net/java/dev/jna/jna/5.6.0/jna-5.6.0.jar \
+    # XXX TODO: should check a sha256sum or something here...
+    && sudo mv jna.jar /opt \
+    && echo "export CLASSPATH=\"\$CLASSPATH:/opt/jna.jar\"" >> /home/circleci/.bashrc \
+    && echo "export CLASSPATH=\"\$CLASSPATH:/opt/jna.jar\"" >> /home/circleci/.profile \
+    && cd ../ \
+    && rm -rf ./setup-jna
```

Comment:
I also added a dockerfile that builds all the tools we need, but I forgot to install JNA on my first attempt, so here it is as part of this PR :-)

If you're curious about the whole file, you can view it [here](https://github.com/rfk/uniffi-rs/blob/becd87c30bceca0af989266ca75633cc7059fd6f/docker/Dockerfile-build).